.github/workflows/build.yml

```diff
@@ -100,7 +100,7 @@ jobs:
 
     - name: Install ReSharper
       run: |
-        dotnet tool install -g JetBrains.ReSharper.GlobalTools --version 2022.2.2
+        dotnet tool install -g JetBrains.ReSharper.GlobalTools --version 2022.3.1
```

Comment:
This should not change. If you want to update resharper in the pipeline, do it in a separate PR, but be aware that it can actually change the way code is formatted. That is why it has been hardcoded.

Reply:
I agree, it was just a test to see if newer version was improving perfs, but not really